setupTests.js

```diff
@@ -12,3 +12,56 @@
 
 // setup file
 import '@testing-library/jest-dom';
+
+const ERROR_PATTERNS_WE_SHOULD_FIX_BUT_ALLOW = [
+  'An update to %s inside a test was not wrapped in act',
+  'Unknown event handler property `%s`',
+  'Not implemented: navigation',
+  'React does not recognize the `%s` prop on a DOM element',
+  'Received NaN for the `%s` attribute',
+  'Can\'t perform a React state update on an unmounted component',
+  '%s contains an input of type %s with both value and defaultValue props',
+  '<%s /> is using incorrect casing',
+  'The tag <%s> is unrecognized in this browser',
+  'The above error occurred in the <TestHook> component'
+];
```

Comment:
Since @snowystinger has addressed a bunch of these in some follow up pulls, should we strip them out of the array or do we want to keep this as is? Fine with it either way

Reply:
I'm happy to update this PR with the minimal set once the PRs from @snowystinger are merged.

Reply:
I wouldn't worry about it in this PR, we can follow up with stripping them out